manuscript.md

```diff
@@ -36,6 +40,36 @@
 
 ## What is a species interaction?
 
+Interactions between species can be manifested in a multitude of ways e.g. different 
+types [@Jordano2016ChaEco], strengths, or symmetry and are not necessarily restricted 
+between only a pair of species e.g indirect interactions [@Morales-Castilla2015InfBio]. The common thread 
+between these multiple forms of interactions is that *at least* one of the species is 
+affected by the interaction, this can be either positively or negatively [@Morales-Castilla2015InfBio]. Combine 
+this high 'diversity' of interaction types with the sheer number of potential 
+interactions that *could* occur in a community and it is understandable why attempting 
+to sample species interaction empirically is a considerable task. However, inferring 
+interactions through proxies such as traits, phylogenies, geographical data and other frameworks 
+could prove to be promising [@Morales-Castilla2015InfBio]. This requires viewing (and constraining?) 
+interaction networks in a variety of ways, such as using graphs or matrices.
+
+In a graph, each species is represented as a node and each interaction as an edge
+[@Delmas2018AnaEco, @Pascual2006EcoNet], thus the value/form an edge takes is a 
+representation of the interaction. When using a matrix format we can uses two 
+approaches, namely bipartite networks where we have two interacting species guilds 
```

Comment:
We can also model tripartite and multipartite networks (e.g. a network of seed, insect seed feeders, and parasitoids). See  for example [Pocock et al. (2012)](https://science.sciencemag.org/content/335/6071/973.abstract?casa_token=mt8c36Ts2fMAAAAA:RPeBuevutBleI7YgYih-15Vv0fIWcUMvMVe3t890wpzIToyYJkdJJni_ltN1ONOEMOfbLLJuVZwuYg) and [Evans et al. (2013)](https://hull-repository.worktribe.com/output/469965/the-robustness-of-a-network-of-ecological-networks-to-habitat-loss).